scripts/acceptance-tests-ide.sh

```diff
@@ -2,46 +2,84 @@
 
 set -euo pipefail
 
-readonly TOOLSMITH_ENV_INPUT="${TOOLSMITH_ENV_INPUT:?must be set}"
-readonly DEFAULT_ORG="${DEFAULT_ORG:?must be set}"
-readonly DEFAULT_SPACE="${DEFAULT_SPACE:?must be set}"
-readonly SKIP_SSL_VALIDATION="${SKIP_SSL_VALIDATION:?must be set}"
-
-declare API_HOST
-readonly API_PORT=443
-readonly USERNAME="admin"
-declare PASSWORD
-readonly CLIENT_ID="admin"
+readonly METADATA_FILE="${METADATA_FILE:?must be set}"
+
+declare CF_API_HOST
+declare CF_USERNAME="admin"
+declare CF_PASSWORD
+
+declare CLIENT_ID="admin"
 declare CLIENT_SECRET
 
+declare DEFAULT_ORG="test"
+declare DEFAULT_SPACE="development"
+
+readonly DEPLOYMENT_DIRECTORY="$(mktemp -d)"
+
 discover_environment() {
   local env_name
-  env_name=$(cat "$TOOLSMITH_ENV_INPUT/name")
+  env_name=$(jq -r .name <"${METADATA_FILE}")
 
-  eval "$(bbl print-env --metadata-file "$TOOLSMITH_ENV_INPUT/metadata")"
+  eval "$(bbl print-env --metadata-file "${METADATA_FILE}")"
 
-  API_HOST="$(jq -r .cf.api_url <"$TOOLSMITH_ENV_INPUT/metadata")"
-  PASSWORD="$(credhub get -n "/bosh-${env_name}/cf/cf_admin_password" -q)"
+  CF_API_HOST="$(jq -r .cf.api_url <"${METADATA_FILE}")"
+  CF_PASSWORD="$(credhub get -n "/bosh-${env_name}/cf/cf_admin_password" -q)"
   CLIENT_SECRET="$(credhub get -n "/bosh-${env_name}/cf/uaa_admin_client_secret" -q)"
 }
 
+prepare_cf_deployment() {
+  pushd "$DEPLOYMENT_DIRECTORY" > /dev/null
+
+  bosh --deployment cf manifest > manifest.yml
+
+  cat << EOF > ops.yml
+- type: replace
+  path: /instance_groups/name=diego-cell/vm_type
+  value: large
+- type: replace
+  path: /instance_groups/name=router/vm_type
+  value: large
+- type: replace
+  path: /instance_groups/name=api/vm_type
+  value: large
+EOF
+
+  bosh --non-interactive --deployment cf deploy --ops-file ops.yml manifest.yml
+
+  popd > /dev/null
+}
+
+prepare_cf() {
+  local -r test_instances_org="$DEFAULT_ORG-instances"
+
+  cf login -a "$CF_API_HOST" -u "$CF_USERNAME" -p "$CF_PASSWORD" -o system --skip-ssl-validation
+
+  cf create-org "$DEFAULT_ORG"
+  cf create-space "$DEFAULT_SPACE" -o "$DEFAULT_ORG"
+
+  cf create-org "$test_instances_org"
+  cf create-space "$DEFAULT_SPACE" -o "$test_instances_org"
+}
+
 print_values() {
-cat <<EOF
-SPRING_CLOUD_APPBROKER_ACCEPTANCETEST_CLOUDFOUNDRY_API_HOST=${API_HOST}
-SPRING_CLOUD_APPBROKER_ACCEPTANCETEST_CLOUDFOUNDRY_API_PORT=${API_PORT}
-SPRING_CLOUD_APPBROKER_ACCEPTANCETEST_CLOUDFOUNDRY_USERNAME=${USERNAME}
-SPRING_CLOUD_APPBROKER_ACCEPTANCETEST_CLOUDFOUNDRY_PASSWORD=${PASSWORD}
+  cat <<EOF
```

Comment:
Whitespace

Reply:
Isn't this good?
```
print_values() {
  echo "We have this"
}

print_values() {
echo "We had this"
} 
 ```
 ```

Reply:
Sorry, my bad! I was thrown by the diff but you're totally right.